src/modules/70-pipeline/components/PipelineInputSetForm/PipelineInputSetForm.tsx

```diff
@@ -217,6 +220,9 @@ export function StageForm({
   stageTooltip?: {
     [key in StageType]?: string
   }
+  repoIdentifier?: string
```

Comment:
```suggestion
  repoName?: string
```

then use same interface here as well

Reply:
Will cleanup in follow up PR